packages/utils/src/base64url.ts

```diff
@@ -41,6 +43,10 @@ export class Base64url {
      *
      */
     public static decode(input: string): Uint8Array {
-        return Uint8Array.from(Buffer.from(input, 'base64url'))
+        return Uint8Array.from(Buffer.from(input, 'base64'))
     }
 }
+
+export function base64ToBase64URL(base64: string) {
```

Comment:
Biggest nit ever. The function does not have to be exported :). Should be the absolute slightest reduction in codegen :').

Reply:
Fixed within the GitHub Mobile app 😎